mvp-spec/audit-log-design.md

```diff
@@ -0,0 +1,257 @@
+                "domain": "cmp1.com",
+                "timestamp": 1639589531,
+                "signature": "12345_signature"
+            }
+        }
+    },
+    "seed": {
+        "version": 0,
+        "transaction_id": "a0651946-0f5b-482b-8cfc-eab3644d2743",
+        "publisher": "publisher.com",
+        "source": {
+          "domain": "ad-server.com",
+          "timestamp": 1639589531,
+          "signature": "12345_signature"
+        }
+    },
+    "transmissions": [
+        {
+            "version": 0,
+            "receiver": "ssp1.com",
+            "status": "success",
+            "details": "",
+            "source": {
+                "domain": "ssp1.com",
+                "timestamp": 1639589531,
+                "signature": "12345_signature"
+            }
+        },
+        {
+            "version": 0,
+            "receiver": "ssp2.com",
+            "status": "success",
+            "details": "",
+            "source": {
+                "domain": "ssp2.com",
+                "timestamp": 1639589531,
+                "signature": "12345_signature"
+            }
+        },
+        {
+            "version": 0,
+            "receiver": "dsp.com",
+            "status": "success",
+            "details": "",
+            "source": {
+                "domain": "dps.com",
+                "timestamp": 1639589531,
+                "signature": "12345_signature"
+            }
+        }
+    ]
+}
+```
+<!--partial-end-->
+
+# Display of the Audit UI
+
+## Web Page interface
+
+Once the user has clicked on the Audit Button, she/he is redirected to the
+Audit UI. The Contracting Party generates the UI based on the validations 
+of the Prebid SSO Data and the Transmission Results. The following elements must
+appear in the page:
+
+| Element                          | Details                                   |
+|----------------------------------|-------------------------------------------|
+| List of Pseudonymous-Identifiers | Each Pseudonymous-Identifier must be paired with the name of the Operator who generated it and signed it with a Reg/Green indicator expressing the validity of the signature.                                                |
+| List of the Preferences          | Each Preference must be paired with the name of the CMP who generated it and signed it with a Red/Green indicator expressing the validity of the signature.                                                             |
+| Seed                             | The Seed is represented by the Transaction ID and a Red/Green indicator expressing the validity of the signature                                                                                                             |
+| List of Transmission Results     | The Transmission Results available in the Audit Log. Each Transmission Results is represented by the Name of the Receiver, the status of the Transmission and a Red/Green indicator expressing the validity of the signature |
```

Comment:
Each Transmission Result~s~

Reply:
Don't get your point. Is it the "s" at the end of the "Results" ?
Fixing it.